libres/lib/enkf/enkf_main.cpp

```diff
@@ -18,14 +18,13 @@
 #include <string.h>
 #include <stdlib.h>
 #include <stdio.h>
-#include <pthread.h>
```

Comment:
Remove

Reply:
You mean put it back? (PR does remove it, afaics)